src/bibliography.jl

```diff
@@ -195,11 +197,15 @@ function parse_bibliography_block(block, doc, page)
     return fields, lines
 end
 
+# TODO: Why is this needed? Wasn't needed in Documenter@0.27 for some reason
+Selectors.matcher(::Type{BibliographyBlock}, x, page, doc) = true
```

Comment:
Did like a fallback got removed in Documenter or something?

Reply:
It looks like `leaf_subtypes` https://github.com/JuliaDocs/Documenter.jl/blob/7c97a86a31e360d7d22082a9a783b0cab24163b5/src/utilities/Selectors.jl#L166C24-L166C24 now includes the type itself, so we hit https://github.com/JuliaDocs/Documenter.jl/blob/7c97a86a31e360d7d22082a9a783b0cab24163b5/src/utilities/Selectors.jl#L168-L169 on master, but not on Documenter 0.27. I am confused then, because we would also end up here https://github.com/JuliaDocs/Documenter.jl/blob/7c97a86a31e360d7d22082a9a783b0cab24163b5/src/utilities/Selectors.jl#L174 so is it called twice now?